code/main/core/identityModule.brs

```diff
@@ -17,146 +17,220 @@ function _adb_isIdentityModule(module as object) as boolean
     return (module <> invalid and module.type = "com.adobe.module.identity")
 end function
 
-function _adb_IdentityModule(configurationModule as object, consentState as object) as object
-    if not _adb_isConfigurationModule(configurationModule) then
-        _adb_logError("IdentityModule::_adb_IdentityModule() - configurationModule is not valid.")
+function _adb_IdentityModule(identityState as object, edgeModule as object, task = invalid as object) as object
```

Comment:
Can you remove task from the function definition?

Reply:
I thought I removed that already. Let me do it.